src/bda/plone/orders/mailnotify.py

```diff
@@ -481,3 +523,11 @@ def notify_booking_cancelled_shopmanager(event):
 
 NOTIFICATIONS['booking_cancelled'].append(notify_booking_cancelled_customer)
 NOTIFICATIONS['booking_cancelled'].append(notify_booking_cancelled_shopmanager)
+
+
+def notify_stock_threshold_reached_shopmanager(event):
+    notify_stock_threshold_reached(event, who="shopmanager")
```

Comment:
Yes, that would be a possibility. you can register notify_stock_threshold_reached directly and call do_notify_shopmanager inside it.